airflow/providers/amazon/aws/example_dags/example_appflow.py

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It's a good question. We are currently working on migrating all the example dags to system tests. Though we just started the projet and it might be a bit too early to ask to implement system tests right away. We have not put down any documentation and there is only a few examples (on AWS side).

On the other hand if @igorborgest wants to go ahead and do it, be my guest :)

Reply:
+1 to vincbeck

I think we're still at the point where we shouldn't need to require that.  If we want to merge it as a sample dag, I can add it to my list of dags that need to be converted.  We only have the one merged and it's not the best sample case, and no real design guide yet (expect one by the end of the week).  If Igor does the conversion now, I certainly won't say no, but we'll likely have to go back and adjust it later anyway.  I'm happy to take on that work.

Reply:
If it is OK with everyone, I prefer not to address that in this PR and handle the conversion in a separate work/PR.